04-gates/00-gates.md

```diff
@@ -2,4 +2,54 @@
 # Protocol gateways
 ## HTTP
 ## S3
+
+[NeoFS S3 gateway](https://github.com/nspcc-dev/neofs-s3-gw) provides API compatible with Amazon S3 cloud storage service.
+
+### Access Box scheme
+
+S3 gateway have to authenticate user requests staying in [AWS spec](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html).
+So we have the following scheme:
+
+![Authmate scheme](pic/authmate-scheme.png)
+
+
+1. User use the [neofs-s3-authmate](https://github.com/nspcc-dev/neofs-s3-gw/blob/master/docs/authmate.md)
+(Authmate) tool to get credentials (`access_key_id` and `secret_access_key`).
+2. Authmate forms "AccessBox" object (see the next section) and put it into NeoFS.
```

Comment:
Authmate forms an "AccessBox" object (see the next section) and puts it into NeoFS.